teamsyncd/teamsync.cpp

```diff
@@ -140,14 +144,23 @@ int TeamSync::TeamPortSync::onPortChange(bool isInit)
         {
             string key = m_lagName;
             key += ":";
+
             uint32_t ifindex = team_get_port_ifindex(port);
             char ifname[MAX_IFNAME + 1] = {0};
-            key += team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+
+            team_ifindex2ifname(m_team, ifindex, ifname, MAX_IFNAME);
+            /* netdev interface is missing */
+            if (ifname[0] == 0)
```

Comment:
i've tested the ifindex, it is not null and the same as before. so this ifindex cannot be used to indicate the interface because it is no longer valid because the netdev is re-generated.